ipatests/test_integration/test_external_ca.py

```diff
@@ -279,3 +279,77 @@ def test_install_external_ca(self):
         # Install new cert
         self.master.run_command([paths.IPA_CACERT_MANAGE, 'install',
                                  root_ca_fname])
+
+
+class TestMultipleExternalCA(IntegrationTest):
+    """setup ext-ca1 install ipa-server with ca1
+     setup ext-ca2 renew ipa-server with ext-ca2
+     check the difference in certificate"""
+
+    def test_master_install_ca1(self):
+
+        install_server_external_ca_step1(self.master)
+        # Sign CA, transport it to the host and get ipa a root ca paths.
+
+        test_dir1 = os.path.join(self.master.config.test_dir, 'CA1')
+        self.master.run_command(['mkdir', test_dir1])
+        # Get IPA CSR as bytes
+        ipa_csr = self.master.get_file_contents(paths.ROOT_IPA_CSR,)
+
+        external_ca = ExternalCA()
+        # Create root CA
+        root_ca = external_ca.create_ca(cn='RootCA1')
+        # Sign CSR
+        ipa_ca = external_ca.sign_csr(ipa_csr)
+
+        root_ca_fname1 = os.path.join(test_dir1, ROOT_CA)
+        ipa_ca_fname1 = os.path.join(test_dir1, IPA_CA)
+
+        # Transport certificates (string > file) to master
+        self.master.put_file_contents(root_ca_fname1, root_ca)
+        self.master.put_file_contents(ipa_ca_fname1, ipa_ca)
+
+        # Step 2 of ipa-server-install.
+        install_server_external_ca_step2(self.master, ipa_ca_fname1,
+                                         root_ca_fname1)
+
+        str1 = "caSigningCert cert-pki-ca"
```

Comment:
Please use a little bit descriptive variable names. E.g. instead of "str1" you can use "ca_nick".